File: covid-api/src/main/java/com/tacs/grupo1/covid/api/dto/CountryListsStats.java
package com.tacs.grupo1.covid.api.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.experimental.Accessors;

import java.util.Date;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Accessors(chain = true)
public class CountryListsStats {
    @JsonProperty("last_days")
    private long lastDays;
    @JsonProperty("start_date")
    private Date startDate;
    @JsonProperty("end_date")
    private Date endDate;
    private long count;
}

File: covid-frontend/src/models/CountryListsStats.ts
export default class CountryListStats{
    public last_days: number;
    public start_date: string;
    public end_date: string;
    public count: number;

    constructor(countryListStats?: CountryListStats){
        if(countryListStats){
            this.last_days = countryListStats.last_days;
            this.start_date = countryListStats.start_date;
            this.end_date = countryListStats.end_date;
            this.count = countryListStats.count;
        }else{
            this.last_days = 0;
            this.start_date = "";
            this.end_date = "";
            this.count = 0;
        }
    }
}